CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 * Updated wdio file path
 * Updated Index template to use terra-doc-template
 * Update template files for new repo file structure
+* Updated template package.json to use js instead of json for jest config
```

Comment:
Hey! I looks like this changelog didn't get updated with the last release `1.9.0` release that occurred on 5/31/18. Can you make this change to reflect that?